radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java

```diff
@@ -49,8 +53,13 @@ private RegisteredValidatorParticle() {
 	}
 
 	public RegisteredValidatorParticle(RadixAddress address, long nonce) {
+		this(address, null, nonce);
+	}
+
+	public RegisteredValidatorParticle(RadixAddress address, String url, long nonce) {
 		super(address.euid());
 		this.address = Objects.requireNonNull(address, "address");

```

Comment:
Should address be null-checked here?

Reply:
Not really necessary, but I think it clarifies intent